packages/space-opera/src/components/shared/section_row/section_row.css.ts

```diff
@@ -18,7 +18,7 @@
 
 import {css, CSSResult} from 'lit-element';
 export const styles: CSSResult = css`:host {
-  color: #dadce0;  /* GOOGLE_GREY_300 */
```

Comment:
I notice we've switched a bunch of greys to blacks and whites. Any particular reason?

Reply:
they're more visually appealing with the current design